src/arrayReverse.test.js

```diff
@@ -7,14 +7,29 @@ describe(`Function 'arrayReverse':`, () => {
     expect(arrayReverse).toBeInstanceOf(Function);
   });
 
-  it(`should return an array`, () => {
-
+  it(`should return an array of reversed worls`, () => {
+    expect(arrayReverse(['Mate', 'Academy'])).toEqual(['ymed', 'acAetaM']);
   });
 
   it(`should return an empty string
     if original array consists of an empty string`, () => {
-    expect(arrayReverse(['Mate', 'Academy'])).toEqual(['ymed', 'acAetaM']);
+    expect(arrayReverse([])).toEqual([]);
+  });
+
+  it(`should return an array of reversed worls including special symbols`, () => {
+    expect(arrayReverse(['&Mate', 'Academy'])).toEqual(['ymeda', 'cAetaM&']);
```

Comment:
The expected result `['ymeda', 'cAetaM&']` does not match the task description. The function should reverse the strings and their order, so the expected result should be `['ymeda', '&etaMca']`.